### ISSUE TITLE
Set z-index of title, subtitle, and other elements

### ISSUE THREAD

**Issue by Ravidelle** [ushelp/EasyQRCodeJS#50]
Would be nice to have an option/property to set the z-index of the title, subtitle, logo, quietzone as to control their layering based on ones needs (and I think the background color of the title isn't working)

**Comment by ushelp** [comment]
Can you provide a description of the scene of z-Index.

Please try the Title test configuration:
```JS
{
  text: "www.easyproject.cn/donation",
  width: 240,
  height: 240,
  
  // Title 
  title: 'Title',
  titleFont: "bold 18px Arial",
  titleColor: "#fff",
  titleBackgroundColor: "#3074B4",
  titleHeight: 40,
  titleTop: 25,
  
  colorDark: "#000000",
  colorLight: "#ffffff",

  
  correctLevel: QRCode.CorrectLevel.H //  L, M, Q, H
}
```

**Comment by Ravidelle** [comment]
My intention was hopefully to utilize the title or subtitle and place it in the foreground of the logo as a fail-safe method to search the particular entry... I wanted it to be part of the generated image... This, of course, could be worked around putting an external element w/ an absolute value but it would be nice if it were integrated in this package...

![image](https://user-images.githubusercontent.com/36964534/116382638-b4d67f80-a848-11eb-99d7-70675a19f301.png)

Basically, in this example, I want the value "#465-229" placed in the blank area I created within the logo... And as you can see, it's placed right behind the logo itself...

**Comment by ushelp** [comment]
Got you now, and good idea. 

Logo image is  asynchronous loading, so the title is drawn before the logo. 

But EasyQRCodeJS is not a suitable foundation, you use unconventional means to achieve your goals. The title, quiet zone, background images, logo are all for common QRCode usage habits. 

If  you want freely control and draw elements, your custom requirements can be implemented in EaysQRCodeJS-Premium:
https://github.com/ushelp/EasyQRCodeJS#easyqrcodejs-premium
